secant/Util/MnemonicSeedPhraseProvider.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  MnemonicSeedPhraseProvider.swift
+//  secant-testnet
+//
+//  Created by Lukáš Korba on 03/09/2022.
+//
+
+import Foundation
+import MnemonicSwift
+
+class MnemonicSeedPhaseProvider: MnemonicSeedPhraseHandling {
```

Comment:
Have you considered replacing the MnemonicSeedPhraseHandling protocol with the usage of the newer simple struct in MnemonicSwift the `MnemonicInteractor`?

Reply:
(just noticed this was previously commented on, sorry for the duplication)